CRM/Core/Payment/PayJunction.php

```diff
@@ -57,6 +57,7 @@ public function __construct($mode, &$paymentProcessor) {
    *   the result in an nice formatted array (or an error object)
    */
   public function doDirectPayment(&$params) {
+    require_once 'PayJunction/pjClasses.php'; // necessary for event registrations 
```

Comment:
Thanks for this. Per our coding standards, comments should go above the line, not on the same line as code. And there should be no trailing space at the end of the line. Thanks!